libcxx/include/__cxx03/__config

```diff
@@ -1218,8 +1218,7 @@ typedef __char32_t char32_t;
 #    define _LIBCPP_DISABLE_UBSAN_UNSIGNED_INTEGER_CHECK
 #  endif
 
-// Clang-18 has support for deducing this, but it does not set the FTM.
-#  if defined(__cpp_explicit_this_parameter) || (defined(_LIBCPP_CLANG_VER) && _LIBCPP_CLANG_VER >= 1800)
+#  if defined(__cpp_explicit_this_parameter)
```

Comment:
Please don't modify the C++03 headers.

Reply:
Good catch, that was not intentionally.

Reply:
@mordante We need Xcode 16.3 in the CI for this change:
https://developer.apple.com/documentation/xcode-release-notes/xcode-16_3-release-notes

Reply:
I missed: https://github.com/llvm/llvm-project/pull/131883 I guess it should be fine now.

Reply:
yes I specifically wrote this patch _after_ #131883 landed; I didn't want more stale cleanup patches that were blocked by old MacOS versions ;-)

Reply:
https://libcxx.llvm.org/#platform-and-compiler-support should be updated with Apple Clang 17.